types/test.d.ts

```diff
@@ -213,7 +213,7 @@ interface ConfigBase {
    * It is possible to pass multiple reporters. A common pattern is using one terminal reporter
    * like `'line'` or `'list'`, and one file reporter like `'json'` or `'junit'`.
    */
-  reporter?: 'dot' | 'line' | 'list' | 'junit' | 'json' | 'null' | ReporterDescription[];
+  reporter?: string | ReporterDescription[];
```

Comment:
It was explicit before so that autocomplete works. Now it will not suggest builtin reporters, unfortunately.

What if we force `reporter: ['my-reporter-module']` for third-party reporters instead?

Reply:
I brought back the autocomplete.